.github/workflows/ci.yml

```diff
@@ -201,7 +201,7 @@ jobs:
         run: cargo test --manifest-path=pyo3-macros-backend/Cargo.toml
 
       - name: Test build config
-        run: cargo test --manifest-path=pyo3-build-config/Cargo.toml
+        run: cargo test --manifest-path=pyo3-build-config/Cargo.toml --all-features
```

Comment:
I'm not sure this is working how I thought it would since codecov is still complaining about the uncovered block in `emit_pyo3_configs()`.

Reply:
Drat - I thought I figured it out but still it's still not running the export-config test in the coverage run.

Reply:
Ha - I think it's "fixed" now but failing windows because the final coverage report command (called by `cargo-llvm-cov`) is very long.